Problem building with native nanopb support

**Environment**
 - OS/Distribution: Debian Bullseye
 - Exact Version or Commit Hash and Installation Method: 6e055d307124b370a98179333fc6cc195ec1fef9

**Describe the bug**
The change introduced in cbfa49551d725b82b89fde665857d67bc907db64 fails the build on my machine. I have installed the nanopb and libnanopb-dev Debian packages, which get detected by the Cmake script, which then fails with an error. It does not occur without those packages installed, as the problematic code section gets skipped.

**To Reproduce**
Steps to reproduce the behavior:

1. Get chiaki 6e055d307124b370a98179333fc6cc195ec1fef9
2. Install dependencies including nanopb and libnanopb-dev
3. Run the cmake procress

**Expected behavior**
The Cmake process finishes without any error messages.

**Log Files**
```
CMake Error at cmake/FindNanopb.cmake:20 (set_target_properties):
  set_target_properties Can not find target to add properties to:
  Jerasure::Jerasure
Call Stack (most recent call first):
  CMakeLists.txt:82 (find_package)
```

--- a/cmake/FindNanopb.cmake
+++ b/cmake/FindNanopb.cmake
@@ -3,9 +3,6 @@
 find_package(nanopb CONFIG)
 find_file(NANOPB_GENERATOR_PY nanopb_generator.py PATH_SUFFIXES bin)
 
-find_path(Nanopb_INCLUDE_DIR NAMES pb_encode.h pb_decode.h)
-find_library(Nanopb_LIBRARY NAMES Nanopb)
-
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(Nanopb
 	FOUND_VAR Nanopb_FOUND
@@ -15,12 +12,7 @@ find_package_handle_standard_args(Nanopb
 )
 
 if(Nanopb_FOUND)
-	set(Nanopb_INCLUDE_DIRS "${NANOPB_INCLUDE_DIR}")
 	if(NOT TARGET Nanopb::nanopb)
 		add_library(Nanopb::nanopb ALIAS nanopb::protobuf-nanopb-static)
-#		set_target_properties(nanopb::protobuf-nanopb-static PROPERTIES
-#		  IMPORTED_LOCATION "${Nanopb_LIBRARY}"
-#		  INTERFACE_INCLUDE_DIRECTORIES "${Nanopb_INCLUDE_DIRS}"
-#		)
 	endif()
 endif()

--- a/cmake/FindNanopb.cmake
+++ b/cmake/FindNanopb.cmake
@@ -18,9 +18,9 @@ if(Nanopb_FOUND)
 	set(Nanopb_INCLUDE_DIRS "${NANOPB_INCLUDE_DIR}")
 	if(NOT TARGET Nanopb::nanopb)
 		add_library(Nanopb::nanopb ALIAS nanopb::protobuf-nanopb-static)
-		set_target_properties(Nanopb::nanopb PROPERTIES
-		  IMPORTED_LOCATION "${Nanopb_LIBRARY}"
-		  INTERFACE_INCLUDE_DIRECTORIES "${Nanopb_INCLUDE_DIRS}"
-		)
+#		set_target_properties(nanopb::protobuf-nanopb-static PROPERTIES
+#		  IMPORTED_LOCATION "${Nanopb_LIBRARY}"
+#		  INTERFACE_INCLUDE_DIRECTORIES "${Nanopb_INCLUDE_DIRS}"
+#		)
 	endif()
 endif()

--- a/cmake/FindNanopb.cmake
+++ b/cmake/FindNanopb.cmake
@@ -3,8 +3,8 @@
 find_package(nanopb CONFIG)
 find_file(NANOPB_GENERATOR_PY nanopb_generator.py PATH_SUFFIXES bin)
 
-find_path(Jerasure_INCLUDE_DIR NAMES pb_encode.h pb_decode.h)
-find_library(Jerasure_LIBRARY NAMES Jerasure)
+find_path(Nanopb_INCLUDE_DIR NAMES pb_encode.h pb_decode.h)
+find_library(Nanopb_LIBRARY NAMES Nanopb)
 
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(Nanopb
@@ -15,11 +15,12 @@ find_package_handle_standard_args(Nanopb
 )
 
 if(Nanopb_FOUND)
+	set(Nanopb_INCLUDE_DIRS "${NANOPB_INCLUDE_DIR}")
 	if(NOT TARGET Nanopb::nanopb)
 		add_library(Nanopb::nanopb ALIAS nanopb::protobuf-nanopb-static)
-		set_target_properties(Jerasure::Jerasure PROPERTIES
-		  IMPORTED_LOCATION "${Jerasure_LIBRARY}"
-		  INTERFACE_INCLUDE_DIRECTORIES "${Jerasure_INCLUDE_DIRS}"
+		set_target_properties(Nanopb::nanopb PROPERTIES
+		  IMPORTED_LOCATION "${Nanopb_LIBRARY}"
+		  INTERFACE_INCLUDE_DIRECTORIES "${Nanopb_INCLUDE_DIRS}"
 		)
 	endif()
 endif()

